client/client.js

```diff
@@ -0,0 +1,28 @@
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { ApolloClient } from 'apollo-client';
+import { HttpLink } from 'apollo-link-http';
+import { ApolloLink } from 'apollo-link';
+
+const cache = new InMemoryCache();
+const httpLink = new HttpLink({
+    uri: process.env.API_URI
+});
+
+const middlewareAuthLink = new ApolloLink((operation, forward) => {
+    const token = "token"
```

Comment:
"token" should not be hardcoded but retrieved via an env variable. Can you remind me why we need this `middlewareAuthLink`?